zooma-mappings-wf/envs/perl-atlas-modules.yaml

```diff
@@ -0,0 +1,6 @@
+channels:
+ - conda-forge
+ - bioconda
+dependencies:
+ - perl
+ - perl-atlas-modules >=0.3.1
```

Comment:
If this is working, this is fine, but I think there is high likelyhood in the future that this will stop to be installable (it is a complex package with way too many dependencies). I would suggest to move to the singularity container for this instead. Specially if the execs calling these are only using perl-atlas-modules. You can either use the `docker://` url or pass the specific file URI in the codon software part with the existing images. This is set at the rule level.

Reply:
I've tried few of the container tags in quay without success. I'll leave the conda env, and open and issue to address this in another PR.